### PR TITLE
finally home

### LINKED ISSUE #2640: in most projects, the user profile leads nowhere
If I'm in, e.g., meinberlin and I have just made myself an account, logged in and nothing else,

and then I click on my username (user indicator) and see my profile,

I see nothing except my name and, in the close future, my description and/or user image.

How do I get back to the process I last looked at?

### PR DESCRIPTION
**Pull request by der-john**
continues #2675.

> wie kommt man zum home view?

Right now, the home view isn't accessible through the UI yet.

In the near future, in the demo project, the home view will be accessible by doing one of the following:
- clicking on the logo on the top left corner (this is currently being implemented in PR #2678 -> #2694),
- having yourself be redirected, e.g. after account activation (`redirect_url` in the config  #2694),
- breadcrumbs in the process header saying "back to process overview",
- breadcrumbs in the user header saying "back to process overview" (this kind-of fixes #2640).

One more thing: the process overview logic is currently done in the `Process` module - and unfortunately includes some custom `meinberlin` code. @xi and I need to consider where the appropriate place for this logic is.

### CODE REVIEW

**Comment by der-john** [comment]
@joka: `collaborative_text` somehow doesn't have  an `IImageReference` sheet - it's not visible in SDI and doesn't get imported via fixtures. Can you look into that?

**Comment by xi** [comment]
Related to #2027

File: src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Process/Process.ts [hunk 1]
```diff
@@ -50,6 +51,16 @@ export var getStateData = (sheet : SIWorkflow.Sheet, name : string) : IStateData
...
 
+var getDate = (utcDate : string) : string => {
+    var date = new Date(utcDate),
+        year = date.getFullYear(),
+        month = ('0' + (date.getMonth() + 1)).slice(-2),
+        day = ('0' + date.getDate()).slice(-2),
+        hour  = ('0' + date.getHours()).slice(-2),
+        minute = ('0' + date.getMinutes()).slice(-2);
+    return day + "." + month + "." + year;
+};
```
Comment (xi) [inline src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Process/Process.ts]: You should use the `adhTime` directive for this instead.

**Comment by xi** [comment]
I propose to split this into several separate steps:

-  [ ] Improve the process list item (first steps in #2692)
  - this might include extending the list item component
-  Improve the process overview introduced in #2027 
   - [ ] choose a nicer layout (603bda8)
   - [ ] Only show a selected list of processes (	c6e7026)
   - [ ] Show some text at the top (8314bfa)
   - [ ] Add an option to `adhListing` to show the total number of elements (37d8545)
   - [ ] Change `adhListing` so you can choose either the traditional list layout or a new "tile" layout.

Also note that he process overview is currently broken, but a fix is available in #2691.

**Comment by der-john** [comment]
It seems to me the remaining todos are basically obsolete.

**Pull request closed without merging**